les/peer.go

```diff
@@ -89,6 +92,7 @@ type peer struct {
 	// RequestProcessed is called
 	responseLock  sync.Mutex
 	responseCount uint64
+	invalidReq    uint64
```

Comment:
Wasn't there some gotcha about atomics that were `64` bits -- or was that only if they were part of a struct? cc @karalabe